File: JAVA PROGRAMMING/Test62.java
// 62. REVERSE NUMBER
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter Input Value: ");
        int num = in.nextInt();

        int rev = 0, NUM = num;

        while (num != 0) {
            int rem = num % 10;
            rev = (rev * 10) + rem;
            num /= 10;
        }
        System.out.println("Reverse Of "+NUM+" Is : "+rev);

        in.close();
    }
}

File: TYPESCRIPT PROGRAMMING/Test62.ts
// 62. REVERSE NUMBER
const Reverse = (num: number): void => {
    let rev: number = 0;
    let NUM: number = num;
    while(num != 0) {
        let rem: number = num % 10;
        rev = (rev * 10) + rem;
        num = Math.floor(num / 10);
    }
    console.log(`Reverse Of ${NUM} Is: ${rev}`);
}
Reverse(123);
Reverse(1234);
Reverse(12345);